### ISSUE TITLE
[WSRM Chxx] Use of "SW Virtualisation Layer" 

### ISSUE THREAD

**Issue by arkadykanevsky** [cntt-n/CNTT#2346]
RM doc uses "SW Virtualisation Layer" to cover both virtualization and containerization SW.
Suggest rewording "SW Virtualisation Layer" into "SW abstraction Layer" or something more encompassing both containerization and virtualization layers.

**Comment by TFredberg** [comment]
I could agree that SW Virtualization is not always a 100% description of a Containerization, but SW Abstraction don´t really describe anything. In most VM and Container deployments you are running your SW application in a separated context from many aspects, but not necessary all aspects. It will likely not be possible to find a 100% correct description that fits both VMs and Containers, very much since they are sometimes used for the same type of virtualization of the underlying resources, and at other times the VMs are subdividing and separating the HW resources and the Containers are more used as a suitable application modularity construct that enable scaling and some desirable Life Cycle Management functions.
Personally I don´t see the benefits of the change.

**Comment by TFredberg** [comment]
By the way, thinking about your comment on ""SW Virtualization" as a concept and name, that was changed way back to be named "Virtualization Infrastructure Layer" that I think better describe the many different types of virtualization concepts implemented by SW. 
What is the document you are referencing [WSRM Chxx]?

**Comment by arkadykanevsky** [comment]
I was referring to RM doc. It is applicable to all chapters.
Should of dropped WS that comes from template.

I view "virtualization" as strict layer that provides support for VMs. Just as "containerization" is a stick layer for container support.
Different technology. And very different application architecture. Trying to come up with a better term that covers both for RM doc.

**Comment by TFredberg** [comment]
Could you point to the exact file you mean, since I have been browsing all chapters and I can not find any chapter heading talking about "SW Virtualization Layer" anymore. We used to have that earlier, but changed it to "Virtual Infrastructure Layer".

**Comment by arkadykanevsky** [comment]
I think saw it at the start of Chapter 3 or 2.

**Comment by pgoyal01** [comment]
In CNTT, aligned with ETSI's use of the term, a virtualised resource can be a VM or container.   From the [CNTT glossary](https://github.com/cntt-n/CNTT/blob/master/doc/common/glossary.md#software-layer-terminology):

**Virtual CPU (vCPU):** Represents a portion of the host's computing resources allocated to a virtualised resource, for example, to a virtual machine or a container. One or more vCPUs can be assigned to a virtualised resource.

[ETSI](gs_NFV-IFA040v040101p) defines an OS Container as: 

**OS container:** virtualisation container utilizing a shared Operating System (OS) kernel of its host NOTE: The host providing the shared OS kernel can be a physical compute node or another virtualisation container.
